Configurations/unix-Makefile.tmpl

```diff
@@ -21,6 +21,35 @@
          return "$target: build_generated\n\t\$(MAKE) depend && \$(MAKE) _$target\n_$target";
      }
      '';
+
+     sub join_filled {
+         my $item_sep = shift;                  # string
+         my $line_sep = shift;                  # string
+         my $first_line_indent = shift;         # number of chars
+         my $line_length = shift;               # number of chars
+
+         my $result = "";
+         my $fill_line = "";
+         my $line_indent = $first_line_indent;
+
+         foreach (@_) {
+             if (length($fill_line . $item_sep . $_)
+                 > $line_length - $line_indent) {
```

Comment:
this line is wrapped.  funny.

Reply:
How do you mean "funny"?  It's broken before an operator...

Reply:
It is amusing that the line to check if wrapping is needed is, itself, line-wrapped because it's too long.   Haha meta-humor.